### PR TITLE
Restructure documentation

### PR DESCRIPTION
**Pull request by KrishnaIyer**
#### Summary

Restructure documentation.

#### Screenshots

Please see the link shared on Slack.

#### Changes

- Change appropriate sections

#### Notes for Reviewers

- All the `aliases` are added so that we don't lose old links. If we do, we can retroactively update it.

#### Checklist

- [ ] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

**Comment by benolayinka** [comment]
Added my changes. @nejraselimovic can you write the "Setup Your First LoRaWAN Network" page in Getting Started? Just walk through creating a discovery account with screenshots.

**Comment by KrishnaIyer** [comment]
@benolayinka: I disagree with some of the restructuring choices in your commit. The idea with the restructuring was to be able to guide users through the docs.

(original) I believe the following  is better than the change.
Screen Shot 2023-04-11 at 13 21 06

(Changed)
Screen Shot 2023-04-11 at 13 21 23

With the new changes, I think that we are creating confusion for new users who end up on the `/gateways` path by throwing new terms at them.

Also the content changes to getting started are just copied from another section and is not clear what the purpose is.

I think it's better if I actually just complete this PR myself. @nejraselimovic: I'll take this over.

**Review by NicolasMrad — APPROVED** [review]

File: doc/content/getting-started/next-steps/_index.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+Visit [The Things Network Learn hub](https://www.thethingsnetwork.org/docs/lorawan/) or [The Things Academy on Udemy](https://www.udemy.com/course/lorawan-fundamentals/) to learn all about LoRa and LoRaWAN, and get prepared for [The Things Fundamentals certification](https://www.thethingsnetwork.org/achievements/a/the-things-certified-fundamentals/).
+
+Check all available certifications on [The Things Certification site](https://www.thethingsnetwork.org/achievements/certificates/).
+
+A lot of academic research around LoRaWAN is being conducted nowadays. [Read some LoRaWAN-related academic papers and publications](https://www.thethingsnetwork.org/docs/lorawan/academic/).
+
+### Useful links
+
+- Check out the [Learn section](https://thethingsnetwork.org/docs/lorawan) of The Things Network website.
+- Read the official specificationa at the [LoRa Alliance](https://lora-alliance.org).
```
Comment (nejraselimovic) [inline doc/content/getting-started/next-steps/_index.md:42]: I would place this content together with the 'LoRaWAN' section in /getting-started, and maybe even make that a separate page ('LoRaWAN basics' or something similar). To me it feels like this doesn't belong here, and the existing /getting-started would be too packed if we just place it there.

File: doc/content/getting-started/_index.md [hunk 2]
```diff
@@ -5,7 +5,88 @@ weight: -2
...
+
+For simple community projects and local testing, there are a few options.
+
+- A free and limited version of {{% tts %}} called [{{% tts %}} Community Edition]{{< relref "ttn" >}} is available to [The Things Network](https://www.thethingsnetwork.org/) community for simple proof-of-concepts and community projects. This service comes with no guarantees and is entirely at the risk of the users.
+
+- For the DIY’ers, the core features of The Things Stack are [open source](https://github.com/thethingsnetwork/lorawan-stack) and is available for local testing.
+
+{{% tts %}} is developed and maintained by [The Things Industries](https://thethingsindustries.com/).
+
+Now that we've covered some basics, let's go ahead and setup your first LoRaWAN network.
```
Comment (nejraselimovic) [inline doc/content/getting-started/_index.md:92]: Seems like an unfavorable position for this (very important for us) content to be at the bottom. What about making this a separate page as well and naming it 'Get familiar with The Things Stack' or something similar, and place the content about creating a discovery network as a subsection of that page? I feel like a majority of the page would just be scrolled over

**Review by nejraselimovic — COMMENTED** [review]
I like the concept at the moment, I have these little remarks though. I think this proposal lacks this content: https://www.thethingsindustries.com/docs/getting-started/what-is-tts/. I know that users complained bunch of times that they hardly differentiate TTI, TTS, TTN and all deployment flavors that we offer

File: doc/content/getting-started/next-steps/_index.md (hunk 1 above)
Comment (KrishnaIyer) [reply doc/content/getting-started/next-steps/_index.md:42]: Ok I've added a "LoRaWAN Basics" section as the first guide.  This content here should belong in this `next-steps` section. We should not overwhelm new readers with all the details right at the start.

**Comment by KrishnaIyer** [comment]
>  I think this proposal lacks this content: https://www.thethingsindustries.com/docs/getting-started/what-is-tts/.

File: doc/content/the-things-stack/migrating/import-devices.md [hunk 3]
```diff
@@ -1,21 +1,17 @@
 ---
 title: Import End Devices in The Things Stack
 weight: 4
-aliases:
-  [
-    "/getting-started/migrating-from-v2/import-devices",
-    "/getting-started/migrating-from-networks/import-devices",
-  ]
+aliases: ["/getting-started/migrating-from-v2/import-devices", "/getting-started/migrating-from-networks/import-devices", /the-things-stack/migrating-from-v2/import-devices, /the-things-stack/migrating-from-networks/import-devices,/getting-started/migrating]
```
Comment (adriansmares) [inline doc/content/the-things-stack/migrating/import-devices.md:4]: Nit: Mixed usage of `"` and non-`"` strings.

File: doc/content/the-things-stack/migrating/migrating-from-v2/_index.md [hunk 4]
```diff
@@ -8,6 +8,7 @@ aliases:
     "/getting-started/migrating-from-v2/configure-ttnctl",
     "/getting-started/migrating-from-v2/export-v2-devices",
     "/getting-started/migrating/migrating-from-v2",
```
Comment (adriansmares) [inline doc/content/the-things-stack/migrating/migrating-from-v2/_index.md:10]: Is this doubled ?

**Review by adriansmares — APPROVED** [review]

**Comment by johanstokking** [comment]
Great stuff!

Few general styling things:

- Use direct link to onboarding page instead of tti.com and then Getting Started
- `literal` is used for many non-code things, like input fields and other things found in the Console, like "`Accounts` page", `Discovery` and `plans`. That should all be **strong** instead
- On `/the-things-stack/interact/console/`, use the tenant ID and region selector.
- On `/getting-started/setup-first-network/`, use (R) for LoRaWAN
- There are a lot of warnings when they don't seem necessary. Example is credit card billing and Helm 3.8 version. Use info or general notice there, or none at all. For prerequisites, we don't use warnings.

> * All the `aliases` are added so that we don't lose old links. If we do, we can retroactively update it.

How can we monitor this?

Please let Anastasia review the main entry pages, including `/getting-started/the-things-stack-basics/` for marketing consistency.

**Review by nejraselimovic — APPROVED** [review]

**Comment by KrishnaIyer** [comment]
> How can we monitor this?

@johanstokking:  https://github.com/TheThingsIndustries/lorawan-stack-docs/pull/1076/commits/03b586734cd40df9fe96d5e8d2f5cf4e60af438d introduces an alias checker. It checks the sitemap of the [currently deployed site](https://www.thethingsindustries.com/docs/sitemap.xml) against the local dev site and compares links.

I'll make this tool automatically run in CI in a follow up PR.

Running the tool for the first time:
```
Original: 690, Unchanged: 657, Aliased: 0, Missing: 33
Missing pages:
- /getting-started/aws/ami/deployment-guide/
- /getting-started/migrating/migration-tool/export-from-tts/
- /getting-started/console/login/
- /getting-started/migrating/migrating-from-v2/major-changes/
- /getting-started/migrating/migrating-from-v2/migrate-using-migration-tool/establish-new-session/
- /getting-started/migrating/migrating-from-v2/packet-broker-requirements/
- /getting-started/console/personal-settings/
- /getting-started/kubernetes/prerequisites/
- /getting-started/console/tenant-settings/
- /getting-started/migrating/migrating-from-v2/migrate-using-migration-tool/migrate-active-session/
- /getting-started/migrating/migrating-from-v2/migrate-using-console/
- /getting-started/migrating/migrating-from-v2/migrate-using-migration-tool/
- /getting-started/aws/ecs/prerequisites/
- /getting-started/aws/ecs/deployment/
- /getting-started/migrating/migration-tool/export-from-chirpstack/
- /getting-started/aws/ami/best-practices/
- /getting-started/aws/ecs/database-operations/
- /getting-started/migrating/import-devices/
- /getting-started/aws/ami/application-server-telemetry/
- /getting-started/aws/ecs/monitoring/
- /getting-started/migrating/configure-mac-settings/
- /getting-started/aws/ami/troubleshooting/
- /getting-started/aws/ecs/updating/
- /getting-started/aws/ecs/best-practices/
- /getting-started/aws/ecs/interop/
- /getting-started/migrating/device-csv/
- /getting-started/aws/ecs/mutual-tls/
- /getting-started/aws/ecs/troubleshooting/
- /getting-started/ttn/addresses/
- /reference/data-formats/
- /getting-started/migrating/migrating-from-v2/migrate-using-takeout-tool/
- /getting-started/aws/ecs/changelog/
- /getting-started/console/troubleshooting/
exit status 1
```

After fixing aliases
```
Check 690 URLs for changes
Original: 690, Unchanged: 690, Aliased: 0, Missing: 0
```

**Review by kschiffer — APPROVED** [review]
Do you have a list of aliases, so we can update the links in the Console as well (without relying on the aliases)?

File: doc/content/getting-started/the-things-stack-basics/_index.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+---
+title: "The Things Stack"
+description: ""
+weight: 2
+aliases: [/getting-started/what-is-tts/]
+---
+
+{{% tts %}} is a LoRaWAN® Network Server stack which is the critical component for any LoRaWAN solution. This guide quickly gets the read up to speed with the basics of {{% tts %}}.
```
Comment (benolayinka) [inline doc/content/getting-started/the-things-stack-basics/_index.md:8]:
```suggestion
{{% tts %}} is a LoRaWAN® Network Server stack which is the critical component for any LoRaWAN solution. This guide quickly gets the reader up to speed with the basics of {{% tts %}}.
```

**Review by benolayinka — APPROVED** [review]
Looks good to me. Thanks for doing this, Krishna.

**Review by johanstokking — APPROVED** [review]

**Comment by johanstokking** [comment]
Great stuff @KrishnaIyer !

**Pull request merged**